quinn/src/connection.rs

```diff
@@ -344,6 +357,9 @@ where
     }
 
     /// Data conveyed by the peer during the handshake, including cryptographic identity
+    ///
+    /// Since the handshake is incomplete at this point, the returned data is likely to be
```

Comment:
Should not be there on `Connection`, only for `Connecting`.

Reply:
Oops, yes sure.

Reply:
Fixed